pkg/asset/installconfig/aws/ec2.go

```diff
@@ -5,11 +5,37 @@ import (
 	"fmt"
 	"time"
 
+	cfgv2 "github.com/aws/aws-sdk-go-v2/config"
+	ec2v2 "github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
+// GetRegions get all regions that are accessible.
+func GetRegions(ctx context.Context) ([]string, error) {
+	// Create a basic/default config. The function is currently called during the survey.
+	// Pass the default region (used for survey purposes) as the region here. Without a region
+	// the DescribeRegions call will fail immediately.
+	cfg, err := cfgv2.LoadDefaultConfig(context.Background(), cfgv2.WithRegion("us-east-1"))
```

Comment:
this function accepts a context so we can use that here instead of context.Background